.env.example

```
@@ -1,3 +1,4 @@
 NOTION_TOKEN=secret_...
 NOTION_DATABASE_ID=...
 PORT=3000
+ALLOWED_EXTENSION_ORIGINS=chrome-extension://<your-extension-id>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALLOWED_EXTENSION_ORIGINS key should go before the NOTION_DATABASE_ID key

(UnorderedKey)